client-app/modules/purchase-requests/components/from-file.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <VcWidget :title="!isMobile ? $t('purchase_request.from_file.title') : undefined" size="md">
+    <div :class="{ 'grid gap-4 md:grid-cols-[auto_290px]': !creatingPurchaseRequest }">
```

Comment:
use BEM

Reply:
Postponed as require a lot of additional time